setup.cfg

```diff
@@ -0,0 +1,13 @@
+[yapf]
+based_on_style = pep8
+blank_line_before_nested_class_or_def = true
+split_before_expression_after_opening_paren = true
+
+[isort]
+line_length = 79
+multi_line_output = 0
+extra_standard_library = setuptools
+known_first_party = mmgpt
+known_third_party = PIL,cv2,mmcv,numpy,pytest,mmengine,torch
+no_lines_before = STDLIB,LOCALFOLDER
+default_section = THIRDPARTY
```

Comment:
seems this file should have an empty line in the end?

Reply:
Thanks for pointing that out. I've added an empty line at the end of the file as suggested. Please let me know if there are any other changes needed.